src/client/modals/cluster-seed-modal/cluster-seed-modal.scss

```diff
@@ -16,8 +16,13 @@
 
 @import '../../imports';
 
-.remove-modal {
```

Comment:
Go home Github, you're drunk. We haven't magically transformed `remove-modal` into `cluster-seed-modal`.